tests/bulk/bulkwrite-update_error-002.phpt

```diff
@@ -28,10 +28,10 @@ echo throws(function() use ($bulk) {
 <?php exit(0); ?>
 --EXPECT--
 OK: Got MongoDB\Driver\Exception\InvalidArgumentException
-update document contains invalid key: empty key
+invalid selector for update: empty key
```

Comment:
Note: this expected output will need to be changed again once https://github.com/mongodb/mongo-c-driver/pull/472 is merged.

Reply:
Do we care enough about the exact word? I'd be happy if we just just %s for the word "selector"/"argument".

Reply:
I personally don't care, but without this test we wouldn't have caught the mistake in libmongoc. I don't think libmongoc is going to add tests for all of its error messages, so I'm OK with keeping this around.